### PR TITLE
build: add deno build system

### PR DESCRIPTION
**Pull request by nekowinston**
I want to be able to override the colors :upside_down_face:

### CODE REVIEW

**Comment by nekowinston** [comment]
demo: https://github.com/nekowinston/dotfiles/commit/1b6f5c775fe6473595ecdb44a88220df0e03a0e7#diff-464b654a364b5079ea0af507cb18e8c299f612f633b971258f91c1335f2e53d3R43-R52

File: deno.json
Comment (ruru-m07) [inline deno.json:1]: a empty file. need to remove.

**Review by ruru-m07 — CHANGES_REQUESTED** [review]
remove it 👍

File: deno.json
Comment (nekowinston) [reply deno.json:1]: was intentional but added `{}`to make it explicit.

**Review by ruru-m07 — APPROVED** [review]
👍

**Pull request closed without merging**